src/renderer/pages/project/project.vue

```
@@ -48,231 +48,242 @@
-                  :isValidScriptData="isValidScriptData"
-                  @update:isValidScriptData="(val) => (isValidScriptData = val)"
-                  @generateImage="(val) => generateImage(val)"
-                  @generateAudio="(val) => generateAudio(val)"
-                  @formatAndPushHistoryMulmoScript="formatAndPushHistoryMulmoScript"
-                  :audioFiles="audioFiles"
-                  :mulmoError="mulmoError"
+        <!-- 3 Split Layout -->
+        <div class="grid grid-cols-1 lg:grid-cols-[30%_40%_1fr] gap-4 h-[calc(100vh-250px)]">
+          <!-- Left Column - AI Chat -->
+          <div class="h-full overflow-y-auto pr-2">
+            <Card :class="`bg-gradient-to-r from-blue-50 to-indigo-50 border-blue-200 h-full flex flex-col ${getTimelineFocusClass}`">
```

ESLint

[error] Replace `·:class="`bg-gradient-to-r·from-blue-50·to-indigo-50·border-blue-200·h-full·flex·flex-col·${getTimelineFocusClass}`"` with `⏎··············:class="`bg-gradient-to-r·from-blue-50·to-indigo-50·border-blue-200·h-full·flex·flex-col·${getTimelineFocusClass}`"⏎············`

(prettier/prettier)